docker-compose.yml

```diff
@@ -1,9 +1,9 @@
 nginx:
-  image: docker.epages.com/epages/docs-nginx:master
+  image: docker.epages.com/epages/docs-nginx:stable
```

Comment:
The image tag should stay named `master` because the rollout dependends on the tags `master` and `develop`.